src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Fruit> fruits = new HashMap<>();
```

Comment:
Using a `Map<String, Fruit>` for the storage might not be the best choice if you need to store quantities of fruits. If you only need to track the quantity, a `Map<Fruit, Integer>` would be more suitable. Make sure the `Fruit` class has properly overridden `equals()` and `hashCode()` methods if it's going to be used as a key in a `HashMap`.